blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
+- `guid`: String; unique id
```

Comment:
Isn't it used by the list component rather than the button itself? (The point being that there's no need for a discriminator value when clicking a button since there's only thing that makes sense to have clicked.)